### ISSUE TITLE
Frontend: contrast of links fail accessibility WCAG 2.1 standard

### ISSUE THREAD

**Issue by brierjon** [openzim/wp1#199]
Running WebAIM and Chrome Lighthouse Accessibility audit both report contrast issues with the color used for links.

WAVE extension can be run against the page to see elements with issues - https://wave.webaim.org/extension/ 
Accessibility standard - https://www.w3.org/TR/WCAG21/#contrast-minimum

**Comment by kelson42** [comment]
@brierjon I think you have reported your ticket in the wrong repository

**Comment by kelson42** [comment]
@brierjon If no, which page of the WP1 engine you complain about exactly? I think a screenshot would be helpful.

**Comment by brierjon** [comment]
@kelson42 Definitely the right repo, I'm also the one who posted the missing prefixes in issue 198.  This accessibility issue applies to https://wp1.openzim.org/#/project/Lakes main table, and table lists.

WebAIM report on the table grid and list:
![image](https://user-images.githubusercontent.com/717866/84538289-adba8780-acf1-11ea-9af6-4c7480484805.png)
![image](https://user-images.githubusercontent.com/717866/84538328-bf9c2a80-acf1-11ea-9ad2-79eadefa5d56.png)

**Comment by kelson42** [comment]
@brierjon Thank you very much
